ngraph_bridge/ngraph_builder.cc

```diff
@@ -2237,35 +2237,76 @@ static Status TranslateGatherV2Op(
   // the full backend creation string
   auto config_map = BackendManager::GetBackendAttributeValues(backend_name);
   if (config_map.at("ngraph_backend") != "NNPI") {
-    return errors::Internal("In translating GatherV2 op ", op->name(),
-                            " found requested backend ", backend_name,
-                            " which is unsupported");
-  }
+    // This workaround works only if axis == 0
```

Comment:
Can we not support Gather for all valid values of axis using nGraph ops Gather or GatherND?

Reply:
I wasn't aware of `Gather` op in nGraph core (I couldn't find it previously), but I rewrote code to use it and all seems to be working fine.

Reply:
I have a small problem in python tests from tensorflow (I think). There is a case (file `build_cmake/artifacts/tensorflow/python/kernel_tests/gather_op_test.py`, test `testBadIndicesCPU`) that expects to catch an error from an op in a form of a string that contains `"indices\[0,0\] = 7 is not in \[0, 2\)"` message. However the nGraph implementation throws a different error and the whole test case fails. How should I handle such test cases?